packaging_automation/update_package_properties.py

```diff
@@ -354,3 +354,5 @@ def update_all_changes(github_token: non_empty(non_blank(str)), package_properti
         create_pr(arguments.gh_token, pr_branch, commit_message, REPO_OWNER, PROJECT_NAME,
                   project.value.packaging_branch)
         remove_cloned_code(execution_path)
+    if not arguments.is_test and not arguments.pipeline:
+        remove_cloned_code(execution_path)
```

Comment:
we already remove this directory in line:356 ? why do we need this change ?

Reply:
I will remove from 356